webpack.config.ts

```diff
@@ -470,14 +479,24 @@ const appConfig: Configuration = {
       chunks: 'async',
       maxInitialRequests: 10, // (default: 30)
       maxAsyncRequests: 10, // (default: 30)
-      cacheGroups: {
-        ...localeChunkGroups,
-      },
+      cacheGroups,
```

Comment:
👏🏼

Reply:
I was doing this for documentation purposes. Changing the name of this variable loses the information that it was maintaining the lits of chunk groups used for locale-based chunking

Reply:
The previous code was making a copy of the object. `cacheGroups: localeChunkGroups` could be the correct approach, but I'd argue for why not include a comment before the variable declaration but indeed I'd go for non-copying solution.

Reply:
Yeah, I think the documentation purposes this served outweighed whatever negligible memory benefits we're gaining here.